platform/viewer/public/config/default.js

```diff
@@ -24,6 +24,11 @@ window.config = {
     prefetch: 25,
   },
   // filterQueryParam: false,
+  /* Dynamic config allows user to pass "configUrl" query string this allows to load config without recompiling application. The regex will ensure valid configuration source */
+  dangerouslyUseDynamicConfig: {
+    enabled: false,
+    regex: /.*/,
```

Comment:
Please provide an example regex that queries locally only, eg:
```
regex: /(0-9A-Za-z.]+)(\/[0-9A-Za-z.]+)*/
That allows numbers and letters in an absolute or sub-path, but not any absolute, with a comment saying that this is a local URL only.  Might have an example showing other specific endpoints, eg:
regex: /(https:\/\/hospital.com(\/[0-9A-Za-z.]+)*)|(https:\/\/othersite.com(\/[0-9A-Za-z.]+)*)/
which restricts to either hosptial.com or othersite.com.  That is the entire point of the restriction that it is possible to choose a specific source of configuration only.

Reply:
@wayfarer3130 will below example works?

```
// regex will ensure valid configuration source and default is /.*/ which matches any character. To use this, setup your own regex to choose a specific source of configuration only.
// Example 1, to allow numbers and letters in an absolute or sub-path only.
// regex: /(0-9A-Za-z.]+)(\/[0-9A-Za-z.]+)*/
// Example 2, to restricts to either hosptial.com or othersite.com.
// regex: /(https:\/\/hospital.com(\/[0-9A-Za-z.]+)*)|(https:\/\/othersite.com(\/[0-9A-Za-z.]+)*)/
```

Reply:
can we add documentation inside https://v3-docs.ohif.org/configuration/configurationFiles as well
Discussing the security concerns and also link to this PR?

Reply:
Yes @sedghi, I've raised PR to @Ouwen for "how to use dangerouslyUseDynamicConfig in configuration documentation".